gui/workorder-recheck.lua

```diff
@@ -0,0 +1,22 @@
+-- Makes selected `Active` work order `Checking`.
+--[====[
+
+gui/workorder-recheck
+=====================
+Sets the status to ``Checking`` (from ``Active``) of the selected work order (in the j-m or u-m screens). This makes the manager reevaluate its conditions.
+
+Example keybinding (put it in your ``dfhack*.init``-file):
+ ``keybinding add Alt-A@jobmanagement/Main gui/workorder-recheck``
```

Comment:
If you want this keybinding in `dfhack.init-example`, I would be totally fine with that, since it's scoped to a single screen. Doing so would also make the keybinding show up here automatically in the docs, so you wouldn't need the example command. It would need to be done in the DFHack repo, so I could handle it separately after merging this. Let me know what you think.

I might suggest `Alt-R`, though, to go along with the name of this script.

As a general comment, changing to this would create a standalone code block, which is generally what we use for example commands (not necessary if we move it to dfhack.init-example, though):

```suggestion
Example keybinding (put it in your ``dfhack*.init``-file)::

    keybinding add Alt-A@jobmanagement/Main gui/workorder-recheck``
```

Reply:
Didn't know about `dfhack.init-example`, so please do add this keybinding there. And yes, Alt-R is probably better.